### ISSUE TITLE
[ENHANCEMENT] Set proper request memory for `virt-api` and `virt-controller` to eliminate related prometheus alerts

### ISSUE THREAD

**Issue by w13915984028** [harvester/harvester#3848]
**Is your enhancement related to a problem? Please describe.**

After a workaround for https://github.com/harvester/harvester/issues/3792,the `KubeVirtComponentExceedsRequestedMemory` Alerts are observed against `virt-api` and `virt-controller`.

**Describe the solution you'd like**

Set proper request memory for those 2 components, instead of the default value.
https://github.com/harvester/harvester/issues/3792#issuecomment-1532699770

**Describe alternatives you've considered**

**Additional context**

issue: https://github.com/harvester/harvester/issues/3792

the commit to set resource for kubevirt: 
https://github.com/harvester/harvester/commit/90ec0db002620706249128c1fb56027130dd3453

**Comment by w13915984028** [comment]
v1.1.2 set the resource limit of kubevirt in https://github.com/harvester/harvester/issues/3427

**Comment by w13915984028** [comment]
Harvester has set resources limits for kube-virt related pods.

https://github.com/harvester/harvester/blob/ad8c9fa3e0700c082ccd8bee441887b49004af51/deploy/charts/harvester/values.yaml#L97

```
        - patch: '{"spec":{"template":{"spec":{"containers":[{"name":"virt-api", "resources":{"limits":{"cpu":"400m","memory":"1100Mi"}}}]}}}}'
          resourceName: virt-api
          resourceType: Deployment
          type: strategic
        - patch: '{"spec":{"template":{"spec":{"containers":[{"name":"virt-controller", "resources":{"limits":{"cpu":"800m","memory":"1300Mi"}}}]}}}}'
          resourceName: virt-controller
          resourceType: Deployment
          type: strategic
        - patch: '{"spec":{"template":{"spec":{"containers":[{"name":"virt-handler", "resources":{"limits":{"cpu":"700m","memory":"1600Mi"}}}]}}}}'
          resourceName: virt-handler
          resourceType: DaemonSet
```

**Comment by w13915984028** [comment]
Close as finished.

**Comment by innobead** [comment]
Is this something already implemented in 1.1 instead of 1.3, right? If yes, we should remove this from 1.3.0 to prevent any confusion.

**Comment by w13915984028** [comment]
From Harvester v1.1.2, the limits were set:
https://github.com/harvester/harvester/blob/21c095a35d7ea93b51136d15cef4c8a635c7e5e8/deploy/charts/harvester/values.yaml
